practices/development.md

```diff
@@ -2,105 +2,90 @@
+In this meeting we prioritize and assign the items that each team member will work on for the next week, and review items that require discussion and planning.
+At the end of each week, we should have completed all of the major delivearables set out for that week.
+  
 (coordination:deliverables)=
 ## Deliverables
 
-Deliverables represent incremental amounts of value we can deliver to a particular stakeholder, and should be completable in a week or two.
+Deliverables represent incremental amounts of value we can deliver to a particular stakeholder, and should be completable in a week.
+They are **encoded as GitHub Issues**.
 
-Most issues in our repositories are deliverables, in varying states of readiness. When a deliverable is first created, it may lack information, be improperly scoped, or have an unclear path to implementation. We improve this through _deliverable refinement_ (see below).
+Most issues in our repositories are deliverables, in varying states of readiness.
+Deliverables that are added to the [Activity Board](coordination:activity-board) should have enough context and tasks that a team member can realistically close them on their own. [^invest]
 
-A deliverable is ready to work (and can thus be added to the Project Backlog) when it has the following properties (adapted from the [INVEST methodology](https://agileforall.com/new-to-agile-invest-in-good-user-stories/)).
+[^invest]: A good resource for considering what kinds of information makes a deliverable "ready" is [the INVEST methodology](https://agileforall.com/new-to-agile-invest-in-good-user-stories).
 
+:::{admonition} Deliverables should...
+:class: tip
 - **Have a short title and description:** Deliverables should be glanceable and have enough context that the reader can quickly understand the scope.
 - **Have one or more [user stories](https://www.atlassian.com/agile/project-management/user-stories):** User stories should define who benefits from the deliverable, and why they want it.
-- **Have completion criteria:** We have clear completion criteria for this deliverable to denote it as “done”.
 - **Have tasks to complete it**: Deliverables should have a set of tasks, which are actions needed to complete the deliverable.
+:::
 
-### How refinement happens
+(coordination:activity-board)=
+## The Activity Board
 
-_Deliverable Refinement_ is the process of improving the scoping, context, and structure of our Deliverables issues so that they are ready for us to work on them. When a deliverable is created, it may not have all of the information needed to take the next step. Adding that information is the goal of Deliverable Refinement.
+The [Activity Board](https://github.com/orgs/2i2c-org/projects/5?fullscreen=true) contains the collection of [Deliverables](coordination:deliverables) that the team is currently working on.
+All deliverables on the Activity Board should be accomplished by the end of the week.
```

Comment:
Again a reference to the end of the week...
This is actually pushing me to think that we are duplicating some stuff with the team sync issue and the deliverables meeting... I mean, the reason to put the deliverable meeting on Tue was having some time on Mon to read through the team sync issue. But the team sync issue has a back / record piece and a next (todo) piece that should be completed after the deliverable meeting. So there is a conflict there...
One way to solve it is just to **remove** the todo part from the team sync issue and trigger that one on Fri morning. Then, we can move the deliverable meetings to Mon and all things align, thoughts?

Reply:
Good point that we're duplicating some information - I think moving our planning meeting to weekly does now duplicate some actions with the deliverables sync.

So you're proposing something like:

- Monday: Have sprint planning meeting, set deliverables to accomplish this week.
- Friday: Have a team sync issue where team members report on progress for their items. Focus this issue more around "what are the major things I did, where did I encounter challenges, and what are items I couldn't get to". Remove the "what will I do next" steps because we'll cover this in the next sprint planning meeting.
- The next Monday: we can focus that meeting around planning next steps rather than covering "what happened last week" because people have already filled out their issue.

Is that right?

Reply:
Yep, indeed, that was my proposal :wink: